i3blocks_mpris.py

```diff
@@ -171,8 +171,9 @@ def __init__(self, bus_name, config=None):
         self._last_info = None
         self._last_status = None
         self._last_metadata = None
-        # a dict set of well-known names with unique instance suffixes
-        self._instances = set()
+        # a dict used as an ordered set, keys — well-known names with unique
+        # instance suffixes, values — True
+        self._instances = {}
```

Comment:
Ah! sorry about this part of the change, my understanding was that dictonaries had no guaranteed order! but that was an old thing (prev to 3.7).

https://docs.python.org/3.7/library/stdtypes.html#dict 

At the moment of writting, at the very bottom of this section there is this note:

> Changed in version 3.7: Dictionary order is guaranteed to be insertion order. This behavior was an implementation detail of CPython from 3.6.